scipy/stats/_distn_infrastructure.py

```diff
@@ -1587,11 +1587,12 @@ def pdf(self, x, *args, **kwds):
         args, loc, scale = self._parse_args(*args, **kwds)
         x, loc, scale = map(asarray, (x, loc, scale))
         args = tuple(map(asarray, args))
-        x = asarray((x-loc)*1.0/scale)
+        dtyp = np.find_common_type([x.dtype, np.float_], [])
```

Comment:
Happy to do this if it makes it clearer (not to me,  I'm thoroughly confused by what are np.float vs np.float_ vs np.float64 vs python float which is C double or is it?)